dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -1214,7 +1214,7 @@ CDefaultContextMenu::MapVerbToCmdId(PVOID Verb, PUINT idCmd, BOOL IsUnicode)
         {
             /* The static verbs are ANSI, get a unicode version before doing the compare */
             SHAnsiToUnicode(g_StaticInvokeCmdMap[i].szStringVerb, UnicodeStr, MAX_VERB);
-            if (!wcscmp(UnicodeStr, (LPWSTR)Verb))
+            if (!wcsicmp(UnicodeStr, (LPWSTR)Verb))
```

Comment:
```suggestion
            if (!_wcsicmp(UnicodeStr, (LPWSTR)Verb))
```
(see commit e4930be4ffb31ce529d797cb90c9613bce163062 for more details)